src/test/unit/algorithms/AnomalyTest.cpp

```diff
@@ -36,42 +36,34 @@ using namespace nupic;
 using namespace nupic::sdr;
 
 TEST(ComputeRawAnomalyScore, NoActiveOrPredicted) {
-  SDR active({10});
-  SDR predicted({10});
+  SDR active({10}, {});
```

Comment:
by default, if you don't want any change, just specify {} as the 2nd argument. 
It's not ideal but better than the behind the scenes conversions.